source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -18,6 +18,10 @@ Mixxx Sound Hardware Preferences
 Mixxx mapping
 ~~~~~~~~~~~~~
 
+Since v2.3 the Master knob is intentionally not linked to Mixxx's Main Gain,
+`following NI's suggestions <https://support.native-instruments.com/hc/en-us/community/posts/360005413857-The-Traktor-S2MK3-Low-output-issue-en-us-?page=1#community_comment_360000742537>`__.
+Indeed this knob also sets the hardware gain, and mosts sets require a high setting.
```

Comment:
This should be a short `... versionchanged` note at the top instead.

Reply:
If I understand correctly, the knob shouldn't have been mapped in the first placed because it controls the hardware volume. So I think this can be made less wordy.